src/config/hooks/mock.conf

```diff
@@ -0,0 +1,10 @@
+# Parameters are replaced using python's format.
+# Available parameters for cmdline: hook_name, taskid, task_results_dir
+# Example:
+# cmdline = {hook_name} {taskid} {task_results_dir}
+#
+# Before the preparation of mock environment
```

Comment:
suggest the more general description "retrace environment" because it can be either mock or podman

Reply:
Changed.

Reply:
And the config item itself?

Reply:
Changed with last commit.